src/voucherify-integration/contentfulApi.js

```diff
@@ -0,0 +1,31 @@
+const getEntry = async (entryId) => {
+  const url =
+    process.env.REACT_APP_CONTENTFUL_MODE === 'PROD'
+      ? `https://cdn.contentful.com/spaces/${process.env.REACT_APP_CONTENTFUL_SPACE_ID}/environments/master/entries/${entryId}?access_token=${process.env.REACT_APP_CONTENTFUL_DELIVERY_API_ACCESS_TOKEN}`
+      : `https://preview.contentful.com/spaces/${process.env.REACT_APP_CONTENTFUL_SPACE_ID}/environments/master/entries/${entryId}?access_token=${process.env.REACT_APP_CONTENTFUL_PREVIEW_API_ACCESS_TOKEN}`
+
+  const response = await fetch(url, {
+    method: 'GET',
+  })
+  try {
+    return (await response.json())?.fields || {}
+  } catch (e) {
+    console.log(e)
```

Comment:
So we allow the application to continue?

Reply:
yes, that is not a fatal error. Just some entry was deleted or moved from published to unpublished or something.